packages/stark-build/config/webpack-partial.common.js

```diff
@@ -19,11 +19,14 @@ const HtmlElementsWebpackPlugin = require("html-elements-webpack-plugin");
 const ContextReplacementPlugin = require("webpack/lib/ContextReplacementPlugin");
 const WebpackMonitor = require("webpack-monitor");
 const BundleAnalyzerPlugin = require("webpack-bundle-analyzer").BundleAnalyzerPlugin;
-// const WriteFilePlugin = require("write-file-webpack-plugin");
```

Comment:
This did not seem to be used anymore, @christophercr?

Reply:
Indeed, the same of the other Plugins but we left them there to remind us that there are still some things to do. so I think we should still keep them ;)